android/app-example/src/androidTest/java/com/badoo/ribs/example/rib/lorem_ipsum/LoremIpsumTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.badoo.ribs.example.rib.lorem_ipsum
+
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
+import android.support.test.espresso.matcher.ViewMatchers.withId
+import com.badoo.common.ribs.RibsRule
+import com.badoo.ribs.RibTestActivity
+import com.badoo.ribs.android.ActivityStarter
+import com.badoo.ribs.android.PermissionRequester
+import com.badoo.ribs.core.directory.Directory
+import com.badoo.ribs.dialog.DialogLauncher
+import com.badoo.ribs.example.R
+import com.badoo.ribs.example.app.AppRibCustomisations
+import com.badoo.ribs.example.rib.lorem_ipsum.builder.LoremIpsumBuilder
+import io.reactivex.functions.Consumer
+import org.junit.Rule
+import org.junit.Test
+
+class LoremIpsumTest {
+
+    @get:Rule
+    val ribsRule = RibsRule { buildRib(it) }
+
+    private fun buildRib(ribTestActivity: RibTestActivity) =
```

Comment:
The same, shouldn't it depend on interface?

Reply:
Same answer as above.